db/schema.rb

```diff
@@ -117,6 +117,15 @@
     t.index ["journal_id"], name: "index_journal_entries_on_journal_id"
   end
 
+  create_table "journal_terms", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "journal_id"
+    t.string "canonical_term"
+    t.text "aliases"
```

Comment:
Would be nice to make this an array, or something more structured than just text (which I assume right now it means to be comma-separated?).

Reply:
I'll give it a shot! I have not used array types at the database level before but it shouldn't be too hard.